guides/common/modules/proc_executing-a-remote-job.adoc

```diff
@@ -82,3 +82,17 @@ To execute a remote job with custom parameters, complete the following steps:
 +
 Replace _query_ with the filter expression that defines hosts, for example `"name ~ rex01"`.
 For more information about executing remote commands with hammer, enter `hammer job-template --help` and `hammer job-invocation --help`.
+
+.Switching between Hosts
+
+When you are on a particular host in the {ProjectWebUI}, you can navigate between hosts without leaving the page with the host switcher.
+Click the{nbsp} image:../images/two-opposite-arrows.png[,15]{nbsp} next to the hostname.
```

Comment:
```suggestion
Click the{nbsp} image:common/two-opposite-arrows.png[,15]{nbsp} next to the hostname.
```

if my understanding of this is correct (it might not be!) you don't need to reference the `images` folder, as that is the default for images anyways, but you do need to reference the `common` one.